### ISSUE TITLE
gp_camera_wait_for_event() timeout value has a minimum that is not documented

### ISSUE THREAD

**Issue by notnot** [gphoto/libgphoto2#612]
I am handling the camera events and noticed that the timeout given in gp_camera_wait_for_event() is not honored if the value is too small. If you specify a timeout of 1 ms, the actual timeout is ~300 ms. Timeout values of >- 300 are behaving as they should. I think this should be documented in the API reference?

**Name the camera**
Sony Alpha-A6300 (Control)     usb:001,046

**libgphoto2 and gphoto2 version**
gphoto2         2.5.20         gcc, popt(m), exif, no cdk, no aa, jpeg, no readline
libgphoto2      2.5.20         all camlibs, gcc, ltdl, EXIF
libgphoto2_port 0.12.0         iolibs: disk ptpip serial usb1 usbdiskdirect usbscsi, gcc, ltdl, USB, serial without locking

**To Reproduce**
call gp_camera_wait_for_event() with a timeout of for instance 1 ms, and time how long the call takes.

**Comment by msmeissn** [comment]
yes, currently the interrupt wait is 300ms .. hmm. i will see what I can do , either adjust the doc
